.github/workflows/ci.yml

```diff
@@ -44,4 +44,4 @@ jobs:
         run: make submodules
 
       - name: 🌌 Coq Universe
-        run: opam exec -- make universe
+        run: opam exec -- make universe DUNEOPT=""
```

Comment:
`opam exec` may also be a PITA for us, we can just try `$(opam env)` ?

Reply:
Windows :clown:

Reply:
Oh wait, we can just include it next to it

Reply:
OK, `eval` doesn't work but I guess that works instead. Although I am not really sure what opam exec does other than set the env.